auth/services/notary/notary.go

```diff
@@ -187,15 +187,13 @@ func (n *notary) Configure() error {
 			return err
 		}
 
-		pkiCfg := pkiconfig.Config{
-			MaxUpdateFailHours: 4,
-		}
-
-		n.uziCrlValidator, err = pki.NewValidator(pkiCfg, truststore.Certificates())
+		// seed pkiValidator with uzi certificate chain
+		err = n.pkiValidator.AddTruststore(truststore.Certificates())
```

Comment:
I'm thinking... shouldn't there be multiple PKI validators, depending on the context? When making n2n calls, you have a different truststore as when verifying UZI signatures? Or doesn't it matter?

Reply:
The PKI validator only verifies the revocation status. If the presented certificate is part of the correct chain is (still) determined per use case. On connections this is part of the tls.Config, and in UZI this is validated against the UZI trust config. By merging the validators we centralize the CRL logic, deduplicate CRL checking (currently CRLs are downloaded twice), and make it possible to add a health check to monitor all of this.